### ISSUE TITLE
@SpringViewDisplay annonation and HotswapAgent+DCEVM

### ISSUE THREAD

**Issue by zesetup** [vaadin/spring#184]
Using Java Java 8 update 112, build 8 + DCEVM+Hotswap Agent.

If any Class which extends UI with annotation "@SpringViewDisplay"  and make  code change appear error: 

org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type 'com.vaadin.spring.internal.SpringViewDisplayRegistrationBean' available: expected single matching bean but found 2: com.vaadin.spring.internal.SpringViewDisplayRegistrationBean#0,com.vaadin.spring.internal.SpringViewDisplayRegistrationBean#1

**Comment by rdavono** [comment]
I get the same error with Java 8u112 and Eclipse with JRebel.

My Workaround is to disable SpringViewDisplayPostProcessor (which creates SpringViewDisplayRegistrationBean). These two classes seem to be some kind of optional performance enhancement (?) but are not really necessary because SpringUIProvider.findSpringViewDisplay() has a fallback when there are missing.

**Comment by zesetup** [comment]
How to disable SpringViewDisplayPostProcessor?
Thank you.

**Comment by rdavono** [comment]
Just don't register it. I stopped using @EnableVaadinNavigation because it automatically registered SpringViewDisplayPostProcessor. You will have to declare the other beans of this annotation manually though. For now it is just one:

```
    @Bean
    @UIScope
    public SpringNavigator vaadinNavigator() {
        return new SpringNavigator();
    }

```

**Comment by hesara** [comment]
Fixed for 1.2.1 and 2.0.2
